### PR TITLE
chore: Add missing code fence languages

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Adding the language to the code fences adds syntax highlighting to the samples making them easier to read and can sometimes show syntax errors. The languages can also allow other tools like ESLint/Prettier to format those blocks if they know how to handle them based on the language

### What's being changed:

Adding the code languages that are supported by the current syntax highlight library

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
👋 @nschonni Thanks for opening a PR! Is this still in progress? When you're ready for review, can you fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) and let me know? Then I'll get it triaged for you!

**Comment by nschonni** [comment]
@janiceilene I've checked the checkboxes now

**Comment by janiceilene** [comment]
@nschonni Could you fill out the **Why?** and **What's being changed:** sections to help ensure that this is triaged to the right team(s) for review, thanks!

**Comment by nschonni** [comment]
@janiceilene done

**Comment by chiedo** [comment]
@janiceilene this should be reviewed by someone on the content team! Looks good to me! But with it being a bunch of content changes, I'll defer.

**Comment by janiceilene** [comment]
Thanks @chiedo! I'll triage it for a writer review.

File: content/actions/creating-actions/setting-exit-codes-for-actions.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ For more information, see "[Creating a JavaScript action](/articles/creating-a-j
 
 If you are creating a Docker container action, you can set a failure exit code in your `entrypoint.sh` script. For example:
 
-```
+```shell
```
Comment (rachmari) [inline content/actions/creating-actions/setting-exit-codes-for-actions.md:43]: This code fence actually removes `<condition>` in the rendered output. You'll need to add `{% raw %}` and `{% endraw %}` around the code fence. 

https://docs-772--code-fence-languages.herokuapp.com/en/free-pro-team@latest/actions/creating-actions/setting-exit-codes-for-actions#setting-a-failure-exit-code-in-a-docker-container-action

File: content/admin/enterprise-management/evacuating-a-cluster-node.md [hunk 2]
```diff
@@ -14,40 +14,40 @@ If you're taking a node offline that has any data services (like git, pages, or
...
 2. You'll need to monitor the status of your node while the data is being copied. Ideally, the node shouldn't be taken offline until the copying is complete. To monitor the status of your node, run any of the following commands:  
 
     For Git
-    ```
+    ```shell
     ghe-spokes evac-status
     ```
     For {% data variables.product.prodname_pages %}
-    ```
+    ```shell
```
Comment (rachmari) [inline content/admin/enterprise-management/evacuating-a-cluster-node.md:28]: This code fence doesn't render `<uuid>`. I think we'll need the raw tags around this code fence too. There are several more examples in this file that will need the same treatment.

**Review by rachmari — COMMENTED** [review]
@nschonni I've reviewed the first 13 files in the PR and there are a couple of instances where the code fence removes some information, particularly information in brackets. Can take a look at those instances and any other instances that might be affected and make the appropriate fix? Then I can take a look at this again. 🙏 💚

File: content/admin/enterprise-management/evacuating-a-cluster-node.md (hunk 2 above)
Comment (nschonni) [reply content/admin/enterprise-management/evacuating-a-cluster-node.md:28]: Interesting, I see that to on the preview site now. Is this actually a bug with the rendering library that I should report upstream? It seems odd that it is altering the content of code blocks as `<code class="hljs language-shell">echo "select count(*) from pages_replicas where host = 'pages-server-<uuid>'" | ghe-dbconsole -y</uuid></code>`

**Comment by nschonni** [comment]
Go away stale bot

**Comment by chiedo** [comment]
Coming soon! https://github.com/github/docs/pull/1386

File: content/actions/creating-actions/dockerfile-support-for-github-actions.md [hunk 3]
```diff
@@ -47,20 +47,21 @@ The Docker `ENTRYPOINT` instruction has a _shell_ form and _exec_ form. The Dock
...
 ENTRYPOINT ["sh", "-c", "echo $GITHUB_SHA"]
-````
+```
 
  To supply `args` defined in the action's metadata file to a Docker container that uses the _exec_ form in the `ENTRYPOINT`, we recommend creating a shell script called `entrypoint.sh` that you call from the `ENTRYPOINT` instruction:
 
 ##### Example *Dockerfile*
-```	
+
+```	dockerfile
```
Comment (hubwriter) [inline content/actions/creating-actions/dockerfile-support-for-github-actions.md:64]:
```suggestion
```dockerfile
```

**Review by hubwriter — APPROVED** [review]
As this has been sitting here for a while, I've gone ahead and added the `raw` and `endraw` markers requested by @rachmari.

@rachmari - could you take another look at this? It would be good to get this merged.

@nschonni - Thanks for raising this PR. 👍

**Comment by hubwriter** [comment]
A little investigation needed here before we can merge this. 

The `{% raw %} ... {% endraw %}` markers appear not to be working on the staging deployment (e.g. see step 2 [here](https://docs-772--code-fence-languages.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/evacuating-a-cluster-node)), although it's working fine on a local deployment.

**Comment by hubwriter** [comment]
On investigating this, it seems that this is in fact a rendering issue with our staging deployment on heroku.

See step 2 of this page, where "<uuid>" is rendered correctly in the shell code, without using `{% raw %} ... {% endraw %}`:
https://docs.github.com/en/enterprise-server@2.21/admin/enterprise-management/evacuating-a-cluster-node

Compare this to the staged deployment of this page for the current PR: 
https://docs-772--code-fence-languages.herokuapp.com/en/enterprise-server@2.22/admin/enterprise-management/evacuating-a-cluster-node

**Comment by rachmari** [comment]
@hubwriter I'd say that we could just remove the language from the code examples that aren't rendering properly and we can file an issue for any that we can look into the cause for later. I'm not sure when we'll have time to investigate the cause and file an issue with the upstream library, but at least we can ship the updates that are working for us now.

File: content/actions/creating-actions/setting-exit-codes-for-actions.md [hunk 4]
```diff
@@ -41,11 +41,13 @@ For more information, see "[Creating a JavaScript action](/articles/creating-a-j
 
 If you are creating a Docker container action, you can set a failure exit code in your `entrypoint.sh` script. For example:
 
-```
+{% raw %}
+```shell
```
Comment (hubwriter) [inline content/actions/creating-actions/setting-exit-codes-for-actions.md:45]:
```suggestion
```
```

**Comment by hubwriter** [comment]
I've removed the language from the opening code fence where the code wasn't rendering properly on the output page.

**Comment by hubwriter** [comment]
@nschonni - Thanks again for working on this. Apologies it's taken us so long to get it merged. 
It's going in now! 😃

**Pull request merged**